docs/source/command_line.rst

```diff
@@ -593,6 +593,41 @@ in error messages.
 
     Show absolute paths to files.
 
+.. option:: --ignore-error-codes CODES
+
+    This flag makes mypy ignore all errors with given error codes. Flag accepts
+    error codes as a comma separated list (there should be no spaces after commas).
+    For example, by default mypy would produce the following errors:
+
+    .. code-block:: python
+
+        class Dynamic:
+            def __init__(self, attr: str, value: object) -> None:
+                setattr(self, attr, value)
+
+        magic_builtin  # error: Name "magic_builtin" is not defined
```

Comment:
The comments make this example look kind of busy, since some of the messages are long. Also, it would be nice to include the error codes in the message, as now it's unclear where the error codes come from (`--show-error-codes` is introduced only later).

Here's a suggested change:
* Remove the comments from the example.
* Include full output from a mypy run, without `--ignore-error-codes` flag, but with `--show-error-codes`.
* Show output from another mypy run, now with the the various error codes ignored.